client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -340,11 +354,13 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 
 export default connect(
 	state => {
+		const user = getCurrentUser( state );
 		return {
 			jetpackConnectAuthorize: state.jetpackConnect.jetpackConnectAuthorize,
-			jetpackConnectSessions: state.jetpackConnect.jetpackConnectSessions
+			jetpackConnectSessions: state.jetpackConnect.jetpackConnectSessions,
+			userId: user ? user.ID : null
```

Comment:
yeah, to add userId to the tracks requests... I've removed both that param as this one